src/user_info.rs

```diff
@@ -90,6 +94,31 @@ where
             .and_then(|http_response| self.user_info_response(http_response).into_future())
     }
 
+    ///
+    /// Submits this request to the associated user info endpoint using the specified asynchronous
+    /// HTTP client.
+    ///
+    #[cfg(feature = "futures-03")]
+    pub async fn request_async<AC, C, F, GC, HC, RE>(
+        self,
+        http_client: C,
+    ) -> Result<UserInfoClaims<AC, GC>, UserInfoError<RE>>
+    where
+        AC: AdditionalClaims,
+        C: FnOnce(HttpRequest) -> F,
+        F: Future<Output = Result<HttpResponse, RE>>,
+        GC: GenderClaim,
+        HC: FnOnce(HttpRequest) -> Result<HttpResponse, RE>,
```

Comment:
oops I think this may have been an oversight on my part, but is `HC` used at all? if not, let's remove it here and above